tests/dirent/test_readdir.c

```diff
@@ -28,17 +28,20 @@ static void create_file(const char *path, const char *buffer, int mode) {
 
 void setup() {
   int err;
-  err = mkdir("nocanread", 0111);
+  err = mkdir("working", 0777);
```

Comment:
Is there some reason to change this test to add the `working` prefix everwhere?

Reply:
This test now runs with NODERAWFS as well as MEMFS via the `also_with_noderawfs` decorator in tests/test_core.py . NODERAWFS's initial working directory [is node's process.cwd() instead of VFS root directory](https://github.com/emscripten-core/emscripten/blob/b55a3c35bd3edb3e2d8ec2cf49abbad8c5851180/src/settings.js#L878-L879) so to make [the output]() match between filesystems I make the test run in a new directory. Maybe this belongs in a new test in tests/fs/ instead?

Reply:
I see.  That makes sense.   Perhaps its worth just adding `mkdir('tmp'); chdir('tmp')` to the top of the file so the rest of it remains unchanged?   In particular I think its good to keep the "." in the final  `scandir("."..`